src/backend/src/Tippecanoe.jl

```diff
@@ -0,0 +1,14 @@
+module Tippecanoe
+export tippecanoe
+
+# NB: not a real package yet
+using tippecanoe_jll: tippecanoe
+
+function tippecanoe(source::String, outdir::String)
+    base = replace(basename(source), r"\.geojson$" => "")
+    return tippecanoe() do bin
+        run(`$bin -zg -pC -f $source -o $outdir/$base.mbtiles`)
```

Comment:
```suggestion
        run(`$bin -zg -pC -f $source --output-to-directory=$outdir`)
```

Reply:
While we're modifying this, we probably want to add `--detect-shared-borders`, too, to do topography-aware simplifications (this may get rid of the gaps between zones, if the input geometry is good enough)